pydatastructs/trees/tests/test_heaps.py

```diff
@@ -29,8 +29,6 @@ def test_BinaryHeap():
     assert max_heap.extract().key == 100
 
     expected_sorted_elements = [36, 25, 19, 17, 7, 3, 2, 1]
-    l = max_heap.heap[0].left
-    l = max_heap.heap[0].right

```

Comment:
Why?

Reply:
I might have accidentally removed them. It was showing a warning if I remember correctly

Reply:
I think the first line can be removed safely, though do it in a different PR.

Reply:
The variable is never being used, so idk whats the point of having it.

Reply:
Sure, feel free to remove it in a separate PR.